app/controllers/finders_controller.rb

```diff
@@ -62,7 +62,10 @@ def show_page_variables
     @pagination = pagination_presenter
     @spelling_suggestion_presenter = spelling_suggestion_presenter
     @ab_test_search_component = use_autocomplete? ? "search_with_autocomplete" : "search"
-    @autocomplete_url = ENV.fetch("SEARCH_AUTOCOMPLETE_API_URL", "")
+    @autocomplete_source_url = ENV.fetch(
+      "SEARCH_AUTOCOMPLETE_API_URL",
+      URI.join(Plek.new.website_root, api_search_autocomplete_path(format: :json)).to_s,
```

Comment:
As this route is generated at request time I imagine we don't actually need to use Plek and can instead do: `api_search_autocomplete_url(format: :json)` which I expect would give it increased environment portability.

Having done a search through the code it looks like people have been scared off of using it before (scarred by broken proxy headers?) though as I can't find `_url` usage, but I did find a helper you could use: https://github.com/alphagov/finder-frontend/blob/d185ae1ec9421a755dbc0b0167cc9fddd2e98974/app/helpers/application_helper.rb#L2-L4

Reply:
There is that, however we'd have to amend the autocomplete component to accept relative URLs (as Javascript's `URL` bizarrely can't deal with them by default).

I'd be happy with that, it's just a tradeoff to make!